Para el caso de las variables, utilizamos t-student, donde la hipotesis nula será no significativo para el modelo

In [ ]:
import scipy.stats as ss

In [ ]:
#calculamos el valor critico de F
alpha= 0.05
f1= 3
f2 = 43
print("valor critico de F: " , round(ss.f.ppf(1-alpha,f1,f2), 3))


valor_critico_de_F:  2.822


A partir de los valores de F, R2 y R2 Adj podemos concluir que el modelo
es capaz de explicar cerca del 0.148% de la varianza.
F es signi
cativo (47.222 > 2.822). Luego, se rechaza la hipótesis nula de que los coe
cientes de regresiones son iguales a cero, por lo que al menos
un coe
ciente debe ser distinto de cero.


In [ ]:
alpha= 0.05
#estadisticos de prueba, que estan extraidos del modelo, como referencia
#tgmat = -1.753
#tnpieu = 2.1252
#temp = 1.273
#tgasto = 3.634
#tcalifev = 5.784

p= alpha/2
obs = 47
regresion = 5
gdl= obs-regresion-1
print("valor critico de t=" ,round(ss.t.ppf(1-p,gdl),3))


valor critico de t= 2.02


Si se cumple que, <br>
 el valor del estadistico de prueba ≥ val_cirtico de t<br> Se rechaza la hipotesis nula, y en el caso contrario no se rechaza<br>
En los casos donde se cumple la igualdad es con las variables:
NPEIU, GASTO Y CALIFEV, por lo que ser rechaza la hipotesis nula para estas variables<br>
Para el caso de GMATy EMP acepta la hipótesis nula por lo que su coe
ciente de regresión es igual a cero
